include/RE/R/Renderer.h

```diff
@@ -70,14 +68,67 @@ namespace RE
 		};
 		static_assert(offsetof(RendererData, lock) == 0x2780);
 
+		struct RendererInitOSData
+		{
+			WinAPI::HWND      hwnd;                // 00
+			WinAPI::HINSTANCE instance;            // 08
+			WinAPI::WNDPROC   windowProcFunction;  // 10
+			WinAPI::HICON     icon;                // 18
+			const char*       className;           // 20
+			uint32_t          adapter;             // 28
+			uint32_t          unk2C;               // 2C
+		};
+		static_assert(sizeof(RendererInitOSData) == 0x30);
+
+		struct ScreenSize
+		{
+			uint32_t width;   // 00
+			uint32_t height;  // 04
+		};
+		static_assert(sizeof(ScreenSize) == 0x8);
+
+		struct ApplicationWindowProperties
+		{
+			ScreenSize screenSize;        // 00
+			int        windowX;           // 08
+			int        windowY;           // 0C
+			int        refreshRate;       // 10
+			int        presentInterval;   // 14
+			bool       appFullScreen;     // 18
+			bool       borderlessWindow;  // 19
+			bool       vsync;             // 1A
+		};
+		static_assert(sizeof(ApplicationWindowProperties) == 0x1C);
+
 		class Renderer
 		{
 		public:
 			[[nodiscard]] static Renderer* GetSingleton() noexcept;
 
+			void Init(RendererInitOSData* a_data, ApplicationWindowProperties* windowProps, WinAPI::HWND window);
```

Comment:
Should these functions be made private? I don't see any reason why this and related functions should be called by the user

Reply:
Yeah, that’s probably wise